frontend/device/devicelistener.lua

```diff
@@ -310,15 +310,27 @@ function DeviceListener:onToggleFlashOnPagesWithImages()
     G_reader_settings:flipNilOrTrue("refresh_on_pages_with_images")
 end
 
-function DeviceListener:onSwapPageTurnButtons(show_notification)
-    G_reader_settings:flipNilOrFalse("input_invert_page_turn_keys")
-    Device:invertButtons()
+function DeviceListener:onSwapPageTurnButtons(show_notification, side)
+    local key_name
+    if side == "left" then
+        G_reader_settings:flipNilOrFalse("input_invert_left_page_turn_keys")
+        Device:invertButtonsLeft()
+        key_name = _("Left-side page-turn buttons")
+    elseif side == "right" then
+        G_reader_settings:flipNilOrFalse("input_invert_right_page_turn_keys")
+        Device:invertButtonsRight()
+        key_name = _("Right-side page-turn buttons")
+    else
+        G_reader_settings:flipNilOrFalse("input_invert_page_turn_keys")
+        Device:invertButtons()
+        key_name = _("Page-turn buttons")
+    end
     if show_notification then
         local new_text
         if G_reader_settings:isTrue("input_invert_page_turn_keys") then
-            new_text = _("Page-turn buttons inverted.")
+            new_text = key_name .. " " .. _("inverted.")
         else
-            new_text = _("Page-turn buttons no longer inverted.")
+            new_text = key_name .. " " .. _("no longer inverted.")
```

Comment:
I'm not sure all this may play well with translations for some languages.

Reply:
Huh, that's a fair point.

Reply:
That can stay as it was before. Not a biggie.

Reply:
Almost certainly not. Being clever is rarely good in such matters.